src/dmd/id.d

```diff
@@ -39,6 +39,11 @@ struct Id
     {
         mixin(msgtable.generate(&initializer));
     }
+
+    extern(C++) void deinitialize()
```

Comment:
I think we can drop ``extern(C++)`` for both initialize and deinitialize.
However, it might be best to keep it for now. To verify that it isn't needed.